### DISCUSSION TITLE
Diagramm aktuelle Ertragswerte zusätzlich  im WebInterface anzeigen

### DISCUSSION THREAD

**Discussion by PiTomar** [tbnobody/OpenDTU#1699] [Ideas]
Könnte man das Diagramm vom ESP- Display evtl. auch im WebInterface bereitstellen?
Ich weiß, über MQTT und ähnlichen lässt sich eine Visualisierung umsetzen und die Daten werden aktuell nicht im ESP persistiert. 

Da die Daten aber zumindest im ESP temporär vorliegen, könnte man es ja evtl. auch ins WebInterface bringen ohne extra Komponenten/ Bridge-Lösungen zu benötigen?

**Comment by ghost** [comment]
Hi, 
wenn es der freie Speicher des ESP zulässt, man es will, dazu die Zeit hat und im Gegensatz zu mir zsätzlich auch noch mehr Ahnung, dann schon:
[Link zum Thema in ander Diskussion](https://github.com/tbnobody/OpenDTU/discussions/1504#discussioncomment-8375856).
